### PR TITLE
Fix em_smt_folding_test Test

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-Added customize support as cpu unit and loop in test 
2- Fixed style issue in test

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: cpu/em_smt_folding_test.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def setUp(self):
             self.cancel("Processor is not ppc64")
         smg = SoftwareManager()
         detected_distro = distro.detect()
+
```
Comment (harish-24) [inline cpu/em_smt_folding_test.py]: new line?

File: cpu/em_smt_folding_test.py [hunk 2]
```diff
@@ -55,6 +56,10 @@ def setUp(self):
         tarball = self.fetch_asset('http://liquidtelecom.dl.sourceforge.net'
                                    '/project/ebizzy/ebizzy/0.3'
                                    '/ebizzy-0.3.tar.gz')
+
```
Comment (harish-24) [inline cpu/em_smt_folding_test.py]: here too

File: cpu/em_smt_folding_test.py (hunk 1 above)
Comment (PraveenPenguin) [reply cpu/em_smt_folding_test.py]: sure i will remove this

**Comment by PraveenPenguin** [comment]
@harish-24 pushed the changes please have a look

File: cpu/em_smt_folding_test.py [hunk 3]
```diff
@@ -31,6 +31,8 @@ class SmtFolding(Test):
     Throughput test for SMT folding in presence of swizzle.
     TODO :  add logic to  Revert back  value smt and cpu  state
             as it was before test.
+
+    :avocado: tags=cpu,power,privileged
```
Comment (harish-24) [inline cpu/em_smt_folding_test.py]: Can you add other `em` tests which does not include tags?

Comment (PraveenPenguin) [reply cpu/em_smt_folding_test.py]: yes sure   i will address that in another PR  i need to do rebase of this patch to add that

Comment (harish-24) [reply cpu/em_smt_folding_test.py]: Better to move this commit also in the new PR #1137 as you have already worked on it.

Comment (PraveenPenguin) [reply cpu/em_smt_folding_test.py]: :( sure but require rebase for this PR

**Comment by PraveenPenguin** [comment]
@harish-24 pushed  the changes :)

File: cpu/em_smt_folding_test.py (hunk 1 above)
Comment (harish-24) [reply cpu/em_smt_folding_test.py]: readded by mistake?

**Comment by PraveenPenguin** [comment]
@harish-24 thanks for ACK , merging

**Pull request merged**